cdap-docs/integrations/source/partners/cloudera/configuring.rst

```diff
@@ -184,7 +141,11 @@ When a new compatible CDAP parcel is released, it will be available via the Parc
 
 .. rubric:: Upgrading Major/Minor Release versions
 
-These steps will upgrade from CDAP 2.8.0 to CDAP 3.0.0. (**Note:** Apps need to be both recompiled and re-deployed.)
+Upgrading between major versions of CDAP involves the additional steps of upgrading the CSD, and running the included
+CDAP Upgrade Tool. Upgrades between multiple Major/Minor versions must be done consecutively, and a version cannot be
+skipped unless otherwise noted.  **Note:** Apps need to be both recompiled and re-deployed.
```

Comment:
This statement was from the generalized instruction list.  I can put this clarification in the "3.0 to 3.1" release-specific instructions.  Do we not want this as part of general major upgrade instructions?  (ie, it applied to 2.8 -> 3.0 as well)